### PR TITLE
SISRP-17837 - Make amounts searchable with dollar signs for Billing Activity

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-17837

Also just made `parseAmounts()` a bit more "pure" =)

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/billingController.js [hunk 1]
```diff
@@ -89,11 +89,23 @@ angular.module('calcentral.controllers').controller('BillingController', functio
...
+
   var parseAmounts = function(value) {
-    if (_.isNumber(value)) {
-      return value.toFixed(2);
+    var newValue = value;
+    if (_.isNumber(newValue)) {
+      newValue = newValue.toFixed(2);
     }
-    return value;
+    return newValue;
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/billingController.js]: @davidlimcheng How about ternary operator here?  
```
return _.isNumber(value) ? value.toFixed(2) : value;
```

**Comment by davidlimcheng** [comment]
Amended, thanks @johncrossman!

File: src/assets/javascripts/angular/controllers/widgets/billingController.js [hunk 2]
```diff
@@ -89,11 +89,20 @@ angular.module('calcentral.controllers').controller('BillingController', functio
...
-    return value;
+    if (billingItem.itemBalance) {
+      var itemBalanceSearch = '$' + billingItem.itemBalance;
+      _.set(billingItem, 'itemBalanceSearch', itemBalanceSearch);
+    }
+  };
+
+  var parseAmounts = function(value) {
+    var newValue = value;
+    return _.isNumber(value) ? newValue.toFixed(2) : value;
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/billingController.js]: @davidlimcheng I don't think you need `newValue` at all. Can this be a one-liner?

**Comment by davidlimcheng** [comment]
Makes sense @johncrossman - love the one-liners.

**Comment by johncrossman** [comment]
One-liners are the BEST!  Thx @davidlimcheng

**Pull request merged**